### DISCUSSION TITLE
Quick Reblog keyboard shortcuts

### DISCUSSION THREAD

**Discussion by contentmint** [AprilSylph/XKit-Rewritten#1333] [Suggestion Box]
As I brought up in the Suggestion Box, I was hoping for a function in XKit Rewritten that would allow one to quickly draft/queue/reblog a post using keyboard shortcuts. It seems to still exist in New XKit, but I can't seem to get it to work. Tumblr has a quick reblog (shift+R) and quick queue (shift+D), but they neglected to add in quick drafts. Can we make it happen?

**Comment by marcustyphoon** [comment]
Make sure to send this in as feedback to https://www.tumblr.com/support as well! If Staff agrees and adds it, that's ideal since then everyone gets it! (I think it's probably doable as an XKit feature but I'm not immediately sure exactly how; I would have to play around with it.)

**Reply by contentmint** [reply]
Already done. Thanks!!
